packages/frontend/src/components/Claim/WinBid/WinForm.tsx

```diff
@@ -16,39 +17,41 @@ interface WinFormProps {
 }
 
 export const WinForm = ({ userBid, withdrawalAmount, setView, voucher, setVoucher }: WinFormProps) => {
+  if (!voucher) {
+    return (
+      <Wrapper>
+        <WinBidForm
+          userBid={userBid}
+          withdrawalAmount={withdrawalAmount}
+          setView={setView}
+          voucher={voucher}
+          setVoucher={setVoucher}
+        />
+      </Wrapper>
+    )
+  }
+
   return (
     <>
-      {voucher ? (
-        userBid.claimed ? (
-          <Wrapper>
-            <VoucherForm voucher="0xD69bcE4E8D0929E16" withdrawnBid={userBid.claimed} />{' '}
-          </Wrapper>
-        ) : (
-          <WrapperRow>
-            <WinFormWrapper>
-              <WinBidForm
-                userBid={userBid}
-                withdrawalAmount={withdrawalAmount}
-                setView={setView}
-                voucher={voucher}
-                setVoucher={setVoucher}
-              />
-            </WinFormWrapper>
-            <VoucherFormWrapper>
-              <VoucherForm voucher="0xD69bcE4E8D0929E16" withdrawnBid={userBid.claimed} />
-            </VoucherFormWrapper>
-          </WrapperRow>
-        )
-      ) : (
+      {userBid.claimed || userBid.winType === WinType.Auction ? (
```

Comment:
Why there is a winType checking? I suppose this form should be shown only when bid has already been claimed  and it does not depends on winType.

Reply:
There should be `WinType` check, because if current user won in auction then we should display voucher claiming view without withdraw view. Auction winner cannot withdraw funds so `userBid.claimed` would be always false.

Current view for auction winner:
<img width="698" alt="image" src="https://user-images.githubusercontent.com/51199571/164213498-16cb921c-1fac-4880-a9ba-09dae19149ec.png">
This check makes left side of component disappear as information about win is shown in previous view.